config/initializers/assets.rb

```diff
@@ -49,7 +48,8 @@
                   password_strength
                   proxy_status
                   about
-                  parameter_override)
+                  parameter_override
+                  charts)
```

Comment:
Gotcha. I must have added the file without realizing it was in the list and then removed the other entry when I realized there were 2. Will fix